src/fillTank.test.js

```diff
@@ -1,11 +1,85 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should be fill all tunk', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 10);
+
+    expect(customer.money).toEqual(2680);
+    expect(customer.vehicle.fuelRemains).toEqual(40);
+  });
+
+  it('should be fill all tunk', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 10, 100);
+
+    expect(customer.money).toEqual(2680);
+    expect(customer.vehicle.fuelRemains).toEqual(40);
+  });
 
+  it('should be fill in only what the client can pay', () => {
```

Comment:
Typo in the test description: 'should be fill in only what the client can pay' should be corrected to 'should fill only what the client can pay for'.